apps/condo/domains/banking/access/BankAccount.js

```diff
@@ -3,21 +3,26 @@
  */
 const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
 
+const { checkBankIntegrationsAccessRights } = require('@condo/domains/banking/access/BankIntegrationAccessRight')
 const {
     queryOrganizationEmployeeFor,
     queryOrganizationEmployeeFromRelatedOrganizationFor,
 } = require('@condo/domains/organization/utils/accessSchema')
 
+const { BANK_INTEGRATION_IDS } = require('../constants')
+
 /**
  * BankAccount entity can be read either by:
  * 1. By admin or support
  * 2. Organization employee
  */
-async function canReadBankAccounts ({ authentication: { item: user } }) {
+async function canReadBankAccounts ({ authentication: { item: user }, context }) {
     if (!user) return throwAuthenticationError()
     if (user.deletedAt) return false
     if (user.isAdmin || user.isSupport) return {}
 
+    if (await checkBankIntegrationsAccessRights(context, user.id, [BANK_INTEGRATION_IDS.SBBOL])) return true
```

Comment:
Ideally this call should aware of specific integration.
Now only SBBOL integration in inspected for access rights.

Theoretically, we can filter records by following criteria:

`{ integrationContext: { integration: { integrationAccessRight: { users: { id_in: user.id } } }`

This way User with access right to SBBOL will be able to read SBBOL-related accounts.

But it can be complex to implement.
Optionally.